extensions/azurecore/package.nls.json

```diff
@@ -21,6 +21,7 @@
 	"config.azureAccountConfigurationSection": "Azure Account Configuration",
 	"config.enablePublicCloudDescription": "Should Azure public cloud integration be enabled",
 	"config.enableUsGovCloudDescription": "Should US Government Azure cloud (Fairfax) integration be enabled",
+	"config.enableUsNatCloudDescription": "Should US National Azure cloud integration be enabled",
```

Comment:
Did we ever confirm this is the right title to use?

Reply:
Not sure but I guess we can always change it